_includes/_code/clickbox-c/main.dart

```diff
@@ -0,0 +1,121 @@
+import 'package:flutter/material.dart';
+import 'package:meta/meta.dart';
+
+void main() {
+  runApp(new MyApp());
+}
+
+//---------------------------- ParentWidget ----------------------------
+
+class ParentWidget extends StatefulWidget {
+  @override
+  ParentWidgetState createState() => new ParentWidgetState();
+}
+
+class ParentWidgetState extends State<ParentWidget> {
+  bool active = false;
+
+  void handleClickboxChanged(bool newValue) {
+    setState(() {
+      active = newValue;
+    });
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return new Container(
+      child: new ClickboxC(
+        active: active,
+        onChanged: handleClickboxChanged,
+      ),
+    );
+  }
+}
+
+//----------------------------- ClickboxC ------------------------------
+
+class ClickboxC extends StatefulWidget {
```

Comment:
sorry, just noticed. But are there "clicks" on mobile? Would it make more sense to say "Tap" ?